Header header
uint32 is_obj_to_move 
geometry_msgs/Pose pose 
geometry_msgs/Point dim
